common/test/java/com/couchbase/lite/BaseCollectionTest.kt

```diff
@@ -22,16 +22,18 @@ import org.junit.Before
 import java.util.*
 
 open class BaseCollectionTest : BaseDbTest() {
-    private var testScope: Scope? = null
-    private var testCollection: Collection? = null
+    protected var testCollection: Collection? = null
+    protected var testColName = ""
+    protected var testScopeName = ""
 
```

Comment:
I don't know if you've come across "lateinit".  This might be one place that it was ok to use it.

Reply:
Cool!!!